### ISSUE TITLE
Filter & tags interaction for regularly scheduled reports and 24- and 48- hour reports

### ISSUE THREAD

**Issue by patphongs** [fecgov/fec-cms#3448]
### Summary

**What we're after:** We need to reconsider the interaction for the 24/48 hour reports and what happens when toggling between them and the "Regularly scheduled report form" section below it. Since 24/48 hour reports are not regular reports, we need to look at how the filters need to change so that it makes more sense. 

This work is blocked by front-end functionality in #3463.

Slack convo: https://fecgov.slack.com/archives/C3X3K6EVA/p1579202493068700

Possible solution: 
Modify the Regularly Scheduled Report Form to gray out when the 24- and 48-hour reports toggle is active. This needs to happen so that users are not confused between regularly scheduled form types and 24- and 48-hour reports._

Interaction logic needs to be developed for the IE filters and filter tags when:
- Toggling between "Processed" and "Raw" data
- Toggling between 24- and 48-hour reports and regularly scheduled form types
- What happens to selected filters under "REGULARLY SCHEDULED REPORT FORM".

Once this is figured out, a new implementation ticket should be created.

![Screen Shot 2020-01-06 at 4 09 22 PM](https://user-images.githubusercontent.com/12799132/71849004-fdd85280-309e-11ea-8ebf-7ff52c4037a4.png)

### Related issues

_List any relevant related issue(s)_
- #3011 
- #3370 

**Completion criteria:**
- [ ] Design interaction based on interaction logic needs noted above
- [ ] Consider other interaction logic needs that are not accounted for above
- [ ] Follow-up with front-end with possible implementation
- [ ] Discuss labeling with content team
- [ ] Open followup implementation ticket

**Comment by patphongs** [comment]
This work is not dependent on IE deduping epic. This is a nice to have to avoid confusion between 24-48 hour forms and regularly scheduled forms.

**Comment by JonellaCulmer** [comment]
Pat Phongsvirajati
Yea the filter interaction is wierd, because you get the same thing in Scenario 1 and Scenario 2.
19 replies

Pat Phongsvirajati  22 minutes ago
@Jonella @laura Having the form filters there (F3X and F5) indicates to users that they are able to narrow their results by selecting both F3X and F5. But this isn’t true, because you get the same count by just selecting 24-48 hour. Basically the scenario 1 and scenario 2 I outlined.

Jonella Culmer  20 minutes ago
I think that’s how it works for all most of the filters. If you choose all the options, you’ll get the same results as if you selected none at all. For example if I start with a list of committees and slowly filter by different committee types, by the end I should have the same list I started with. (edited) 
:heavy_plus_sign:
1


Pat Phongsvirajati  17 minutes ago
True, but that’s all within one filter. This is through 2 separate filters in which one that dictates the other. So it’s not clear that both filters are the same in certain circumstances.

Paul Clark  13 minutes ago
They aren't the same. is_notice gives me the 24- and 48- hour reports of IEs, and gives me both F24 (F3X) and F5. The form type filers let me decide if I want to see only F24 (F3X) or only F5.
The interaction might be different. The form types could be selected by default. Then the user could decide if they want leave as is an see all or if they want to uncheck one of the form types.

Pat Phongsvirajati  10 minutes ago
@paul, that’s true, but selecting both form types gives me the same filter as selecting none at all, so it’s kind of counter intuitive that way

Laura Beaufort:spiral_calendar_pad:  8 minutes ago
Maybe we could do some user testing to see if it’s a barrier?

Pat Phongsvirajati  8 minutes ago
I know my above statement can be confusing ^^. I’m just saying that it’s not as intuitive that this is the case. If that’s how we want to build it, then that’s fine, but I find it strange

Paul Clark  8 minutes ago
is_notice means give me all F24s and F5s that are 24- and 48- hour reports of IEs. That's why I suggested that both be checked by default.

Paul Clark  7 minutes ago
Oh, it's completely not intuitive! (edited) 
:heavy_plus_sign:
2


Laura Beaufort:spiral_calendar_pad:  7 minutes ago
@paul that makes sense for this example, but it would break the existing pattern that @Jonella mentioned - we don’t start with all options selected for any other filters

Paul Clark  6 minutes ago
Yes ... I get that. I think we leave the not so intuitive interaction as is, and ask Jonella to do some usability testing on it.
:white_check_mark:
1


Jonella Culmer  5 minutes ago
Yes to usability testing. There’s also a design ticket to address it in 11.3.
:+1:
1


Pat Phongsvirajati  4 minutes ago
Ok thank you for indulging me on the complexity of this logic :simple_smile:
:high_five:
1


Jonella Culmer  3 minutes ago
It may be helpful to add some of what was discussed to the ticket so that whoever picks it up next sprint will have additional info. I don’t think we can link to threads. But that might be cool.

Jonella Culmer  2 minutes ago
Oh, wait you can link to threads!
:woohoo:
1


Laura Beaufort:spiral_calendar_pad:  2 minutes ago
Yeah! Will only be accessible to us though
:100:
1


Pat Phongsvirajati  2 minutes ago
Oh awesome! Can you add it to your design ticket @Jonella?

Pat Phongsvirajati  1 minute ago
Oh that’s true, I can explain my reasoning in the ticket too, so that it’s more open
:heart:
1


Paul Clark  < 1 minute ago
I think you can copy a Slack thread or message and just paste it into a GitHub issue.
:+1:
1

**Comment by patphongs** [comment]
Giving more context to the thread above ^^. An interaction that is not intuitive is that selecting all the report form filters yields the same results as not selecting them at all. 

For example:

The below filter will display all 24- and 48-Hour Reports, which is all form 24(3X) and form 5. 
![Screen Shot 2020-01-16 at 2 49 57 PM](https://user-images.githubusercontent.com/12799132/72557860-94f79400-386f-11ea-827f-68fa5170940b.png)

This filter will yield the same results, selecting 24- and 48-Hour Reports that are both form 24(3X) and form 5. 
<img width="287" alt="Screen Shot 2020-01-16 at 12 26 09 PM" src="https://user-images.githubusercontent.com/12799132/72557773-68437c80-386f-11ea-89ed-ba60bcc8079a.png">

**Comment by patphongs** [comment]
Closing this issue in favor of implementing this solution: #3463
